src/frontends/pytorch/src/op/quantize.cpp

```diff
@@ -0,0 +1,83 @@
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_quantize_per_tensor(const NodeContext& context) {
+    num_inputs_check(context, 4, 4);
+    const auto input = context.get_input(0);
+    const auto scale = context.get_input(1);
+    const auto zero_point = context.get_input(2);
+    const auto dtype = convert_dtype(context.const_input<int64_t>(3));
+
+    const auto scale_convert = context.mark_node(std::make_shared<v1::ConvertLike>(scale, input));
+    const auto zero_point_convert = context.mark_node(std::make_shared<v1::ConvertLike>(zero_point, input));
+    const auto scaled_input = context.mark_node(std::make_shared<v1::Divide>(input, scale_convert));
+    const auto scaled_input_with_zero_pt =
+        context.mark_node(std::make_shared<v1::Add>(scaled_input, zero_point_convert));
+    const auto quantized_input =
+        context.mark_node(std::make_shared<v5::Round>(scaled_input_with_zero_pt, v5::Round::RoundMode::HALF_TO_EVEN));
+
+    ov::Output<ov::Node> output;
+    if (dtype == element::u8) {
+        const auto clamp = context.mark_node(std::make_shared<v0::Clamp>(quantized_input,
+                                                                         std::numeric_limits<unsigned char>::lowest(),
+                                                                         std::numeric_limits<unsigned char>::max()));
+        output = context.mark_node(std::make_shared<v0::Convert>(clamp, element::u8));
+    } else if (dtype == element::i8) {
+        const auto clamp = context.mark_node(std::make_shared<v0::Clamp>(quantized_input,
+                                                                         std::numeric_limits<char>::lowest(),
+                                                                         std::numeric_limits<char>::max()));
+        output = context.mark_node(std::make_shared<v0::Convert>(clamp, element::i8));
+    } else {
+        output = context.mark_node(std::make_shared<v0::Convert>(quantized_input, element::i32));
+    }
+
+    const auto quantized_pt_node =
+        context.mark_node(std::make_shared<QuantizedPtNode>(QuantizedPtNodeType::QUANTIZE_PER_TENSOR,
+                                                            context,
+                                                            output.get_node_shared_ptr(),
+                                                            scale.get_node_shared_ptr(),
+                                                            zero_point.get_node_shared_ptr()));
+    return {quantized_pt_node};
+}
+
+OutputVector translate_quantize_per_channel(const NodeContext& context) {
+    num_inputs_check(context, 5, 5);
+    const auto input = context.get_input(0);
+    const auto scales = context.get_input(1);
+    const auto zero_points = context.get_input(2);
+    const auto axis = context.get_input(3);
+    const auto dtype = convert_dtype(context.const_input<int64_t>(4));
+    // TODO
```

Comment:
This operation should also be implemented using FakeQuantize openvino op.

Reply:
Implemented!